src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -387,33 +387,56 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // GH#8522, GH#3758 - Only modify the selection on key _down_. If we
         // modify on key up, then there's chance that we'll immediately dismiss
         // a selection created by an action bound to a keydown.
-        if (HasSelection() &&
-            !KeyEvent::IsModifierKey(vkey) &&
+        if (!KeyEvent::IsModifierKey(vkey) &&
             vkey != VK_SNAPSHOT &&
             keyDown)
         {
-            // try to update the selection
-            if (const auto updateSlnParams{ ::Terminal::ConvertKeyEventToUpdateSelectionParams(modifiers, vkey) })
+            // TODO CARLOS: should we make this ctrl+shift+m instead?
+            if (modifiers.IsCtrlPressed() && vkey == 'M')

```

Comment:
Whatcha think?

Reply:
make this an action, default it to ctrl+shift+m. The keybindings w/in mark mode don't need to be configurable now, but the toggle should be.